task1.js

```diff
@@ -0,0 +1,119 @@
+const express = require('express');
+const bp = require('body-parser');
+const Joi = require('joi');
+const app=express();
+const port=3000;
```

Comment:
Common practice is to create one place where you will store all config data.
Please, perform the next steps:
1. create a separate file with environment variables it is common practice to name it ".env"
2. move port variable to .env file (all variables in .env have to be in upper register: port -> PORT, someAnotherVariable -> SOME_ANOTHER_VARIABLE)
3. create a new directory "config", create index.js file in the config dir
4. config/index.js will be used to get access to project configs:

example of .env:
`SOME_ANOTHER_VARIABLE=13`

example of config/index.js:
`export const config = {
    someVariable: process.env.SOME_ANOTHER_VARIABLE
}`

When you need a value of SOME_ANOTHER_VARIABLE  in your code, you just import config from config/index.js